### DISCUSSION TITLE
Fonts and assets path

### DISCUSSION THREAD

**Discussion by leads** [cypress-io/cypress#24337] [Questions and Help] [answered]
I can't get on Discord currently so I'll ask here.

In `component-index.html`  I need to link to fonts. It works if I host them externally but that doesn't seem in the spirit of testing, it should be local files. What is the root path for components being tested?

**Comment by stormsson** [comment]
@leads  did you ever find an answer ?

**Comment by jennifer-shehane** [comment, marked as answer]
Local assets must be served by your dev server, and how depends on the bundler:

- Vite: put files in the `public/` directory and reference them root-relative `(url('/fonts/…'))`. Vite serves them under its base path automatically.
- Webpack: `webpack-dev-server` serves no static dir by default, either import the asset so webpack emits it, or add `devServer: { static: { directory: path.join(__dirname, 'public') } }` to your webpack config.

If it still 404s, check the browser network tab to confirm the path resolves. Docs are being updated with this.
